doc/guides/west/manifest.rst

```diff
@@ -219,7 +221,14 @@ The list of project keys and their usage follows. Sometimes we'll refer to the
   :ref:`west-extensions` for details.
 - ``import``: Optional. If ``true``, imports projects from manifest files in
   the given repository into the current manifest. See
-  :ref:`west-manifest-import` for more details.
+  :ref:`west-manifest-import` for details.
+- ``groups``: Optional, a list of groups the project belongs to. See
+  :ref:`west-manifest-groups` for details.
+- ``submodules``: Optional. If present, ``west update`` will also update `Git
```

Comment:
It's a bool or a list, so this sentence is technically not accurate? i.e. `submodules: false`

Reply:
Good catch, thanks. Will change to:

```
You can use this to make ``west update`` also update `Git submodules`_
defined by the project.
```